repotracker/container.py

```diff
@@ -122,7 +122,7 @@ def skopeo_run(reporef, *args):
     Run skopeo with the given args, against the given repo reference.
     Return the CompletedProcess object associated with the skopeo command.
     """
-    cmd = ["/usr/bin/skopeo", *args, f"docker://{reporef}"]
+    cmd = ["/usr/bin/skopeo", "--command-timeout", "60s", *args, f"docker://{reporef}"]
```

Comment:
Out of curiosity, why is the timeout necessary? Have you run into issues with the command getting hung up? Or is this is a preemptive measure?

Reply:
It's preemptive, I haven't seen any timeouts of the `skopeo` command. But it seemed prudent, and for parity with the timeouts when using the quay.io API.